libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -281,6 +284,17 @@ export class PageComponent
   }
 
   ngAfterViewInit(): void {
+    this.contentScrolled$ = this.content.ionScroll.pipe(
+      debounceTime(contentScrollDebounceTimeInMS),
+      map((event) => event.detail),
+      takeUntil(this.ngOnDestroy$)
+    );
+
+    this.contentScrolled$.subscribe((scrollInfo: ScrollDetail) => {
+      this.isContentScrolled = scrollInfo.scrollTop > contentScrolledOffsetInPixels;
+      this.changeDetectorRef.detectChanges();
```

Comment:
Are we aware of any possible performance issues with this?
If I read it correctly, we might get quite a lot of change detection cycles when scrolling.

Reply:
No, the event stream is throttled by a debouce of 10ms so only very few event gets through :)